bramkouwen/Music-Mixer
# Adding PDF to assets

## What does the PR do?
* It adds a project description and note pdf to assets

## How do you check this PR?
* Pull down the branch
*  See added pdf 

## Screenshots 
None.
